File: src/Example3Java/EX04.java
package Example3Java;

import java.util.Scanner;

public class EX04 {
    public static void main(String[] args) {
        System.out.print("Enter number : ");
        String number = new Scanner(System.in).nextLine();
        String str1 = number.substring(0, 2);
        String str2 = number.substring(9);
        System.out.println("year = " + str1 + ", No." + str2);
    }
}

File: src/Example3Kotlin/EX04.kt
import java.util.*

object EX04 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter number : ")
        val number = Scanner(System.`in`).nextLine()
        println("Year = ${number.substring(0, 2)} ,No. ${number.substring(9)}")
    }
}
